addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
Yes, it definitely works. I just don't know if that's _guaranteed_ in all browsers, everywhere, forever. The reason `wait` works at all is because Ember Promises are actually scheduled on the runloop, as are UI Events. RAF steps _outside_ of the normal runloop system, so we're relying on the browser's behavior and I can't figure out where UI Events are scheduled relative to RAF.

We may want to loop @runspired in on this because he's more familiar with how the browser's internal scheduling works.